package.json

```diff
@@ -38,14 +38,11 @@
   },
   "homepage": "https://github.com/ipld/js-ipld-raw#readme",
   "dependencies": {
-    "cids": "~0.8.0",
+    "cids": "^0.8.1",
```

Comment:
Why would you get different versions? In 0.x.0 versions you get the same range with `~` and `^`. I tried at https://semver.npmjs.com/.